tests/storage/snapshot.go

```diff
@@ -845,6 +846,204 @@ var _ = Describe(SIG("VirtualMachineSnapshot Tests", func() {
 					}
 				})
 			})
+
+			Context("online live updates", func() {
+				BeforeEach(func() {
+					virtClient = kubevirt.Client()
+					updateStrategy := &v1.KubeVirtWorkloadUpdateStrategy{
+						WorkloadUpdateMethods: []v1.WorkloadUpdateMethod{v1.WorkloadUpdateMethodLiveMigrate},
+					}
+					rolloutStrategy := pointer.P(v1.VMRolloutStrategyLiveUpdate)
+					err := kvconfig.RegisterKubevirtConfigChange(
+						kvconfig.WithWorkloadUpdateStrategy(updateStrategy),
+						kvconfig.WithVMRolloutStrategy(rolloutStrategy),
+					)
+					Expect(err).ToNot(HaveOccurred())
+
+					currentKv := libkubevirt.GetCurrentKv(virtClient)
+					kvconfig.WaitForConfigToBePropagatedToComponent(
+						"kubevirt.io=virt-controller",
+						currentKv.ResourceVersion,
+						kvconfig.ExpectResourceVersionToBeLessEqualThanConfigVersion,
+						time.Minute)
+				})
+
+				It("should include cpu hotplug in vm snapshot", func() {
```

Comment:
This and the following test could be combined into a single case to save resources? These tests look rather expensive.